### PR TITLE
Add Artemis JMS documentation

### PR DESCRIPTION
**Pull request by middagj**
As requested in #2930 an Artemis guide. I tried to copy AMQP's example, but couldn't got the Server-sent-events working in this short time so I used a simple fetch button.

### CODE REVIEW

File: docs/src/main/asciidoc/artemis-jms-guide.adoc [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+This command generates a Maven project, importing the Artemis JMS extension.
+
+== Starting an Artemis server
+
+Then, we need an Artemis server.
+You can follow the instructions from the https://activemq.apache.org/components/artemis/[Apache Artemis web site] or create a `docker-compose.yaml` file with the following content:
+
+[source, yaml]
+----
+# A docker compose file to start an Artemis server
```
Comment (gastaldi) [inline docs/src/main/asciidoc/artemis-jms-guide.adoc]: Is docker-compose really necessary? I managed to run Artemis JMS using just docker:

```docker run -it --rm -p 61616:61616 -e ARTEMIS_USERNAME=myuser -e ARTEMIS_PASSWORD=otherpassword vromero/activemq-artemis```

Comment (middagj) [reply docs/src/main/asciidoc/artemis-jms-guide.adoc]: No, it isn't, but it was copied from [AMQP guide](https://github.com/quarkusio/quarkus/blob/master/docs/src/main/asciidoc/amqp-guide.adoc#starting-an-amqp-broker)

Comment (middagj) [reply docs/src/main/asciidoc/artemis-jms-guide.adoc]: Also fixed in the corresponding quarkus-quickstarts

File: docs/src/main/asciidoc/artemis-jms-guide.adoc [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+The data can be fetch from a browser using a fetch button from a JAX-RS resource.
+
+== Solution
+
+We recommend that you follow the instructions in the next sections and create the application step by step.
+However, you can go right to the completed example.
+
+Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
+
+The solution is located in the `artemis-jms` {quickstarts-tree-url}/artemis-jms[directory].
```
Comment (gsmet) [inline docs/src/main/asciidoc/artemis-jms-guide.adoc]: OK, we will need a quickstart too then :).

File: docs/src/main/asciidoc/artemis-jms-guide.adoc [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+    @Produces(MediaType.TEXT_PLAIN)
+    public String last() {
+        return consumer.getLastPrice();
+    }
+}
+----
+
+== Configuring the Artemis properties
+
+We need to configure the Artemis connection propreties.
```
Comment (gsmet) [inline docs/src/main/asciidoc/artemis-jms-guide.adoc]:
```suggestion
We need to configure the Artemis connection properties.
```

File: docs/src/main/asciidoc/artemis-jms-guide.adoc [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+Nothing spectacular here. On each fetch, it updates the page.
+
+== Get it running
+
+If you followed the instructions, you should have the Artemis server running.
+Then, you just need to run the application using:
+
+[source, shell]
+----
+mvn compile quarkus:dev
```
Comment (gsmet) [inline docs/src/main/asciidoc/artemis-jms-guide.adoc]:
```suggestion
./mvnw compile quarkus:dev
```

File: docs/src/main/asciidoc/artemis-jms-guide.adoc [hunk 5]
```diff
@@ -0,0 +1,287 @@
...
+
+Open `http://localhost:8080/prices.html` in your browser.
+
+== Running Native
+
+You can build the native executable with:
+
+[source, shell]
+----
+mvn package -Pnative
```
Comment (gsmet) [inline docs/src/main/asciidoc/artemis-jms-guide.adoc]:
```suggestion
./mvnw package -Pnative
```

**Review by gsmet — CHANGES_REQUESTED** [review]
Thanks, I added a few suggestions.

File: docs/src/main/asciidoc/artemis-jms-guide.adoc (hunk 2 above)
Comment (middagj) [reply docs/src/main/asciidoc/artemis-jms-guide.adoc]: I thought that was required :)

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Looks good to me, now, thanks! Let's wait for CI. I let you prepare the quickstart and create a PR for it?

**Comment by middagj** [comment]
Thanks I took over your suggestion. I used several other guides as example, so you might want to change the `mvn` into `./mvnw` there as well.

**Comment by middagj** [comment]
@gsmet, I already created a quickstart, see #quarkusio/quarkus-quickstarts#291

**Comment by gsmet** [comment]
OK, perfect, thanks! I'll have a look soon.

**Pull request merged**